src/js/common/schemaform/definitions/file.js

```diff
@@ -0,0 +1,57 @@
+import _ from 'lodash/fp';
+
+import FileField from '../FileField';
+import { validateFileField } from '../validation';
+
+const defaults = {
+  fileTypes: ['pdf', 'txt', 'jpg', 'jpeg', 'gif'],
+  maxSize: 26214400,
+  endpoint: '/v0/uploads'
+};
+
+export default function fileUiSchema(label, userOptions = {}) {
+  const options = _.assign(defaults, userOptions);
+  return {
+    'ui:title': label,
+    'ui:field': FileField,
+    'ui:options': {
+      endpoint: options.endpoint,
+      fileTypes: options.fileTypes,
+      maxSize: options.maxSize,
+      showFieldLabel: true
+    },
+    'ui:errorMessages': {
+      required: 'You must upload a file',
+      minItems: 'You must upload a file'
+    },
+    'ui:validations': [
+      validateFileField
+    ]
+  };
+}
+
+// An example schema so we don't forget it for now
```

Comment:
👍
I imagine this will get moved into the schemaform docs when it's a little more stable, yeah?